web/services/nft_service.py

```
@@ -0,0 +1,243 @@
+        # Load contract address and ABI
+        self.contract_address = settings.NFT_CONTRACT_ADDRESS
+        if not self.contract_address:
+            raise ValueError("❌ NFT_CONTRACT_ADDRESS is missing in environment variables")
+
+        self.contract_address = self.web3.to_checksum_address(self.contract_address)
+        print(f"📜 Smart Contract Address: {self.contract_address}")
+
+        self.contract_abi = self._load_contract_abi()
+        self.contract = self.web3.eth.contract(address=self.contract_address, abi=self.contract_abi)
+
+    def _load_contract_abi(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_load_contract_abi`

(ANN202)